rero_ils/alembic/21a994dc2beb_add_library_pid_to_ill_request_.py

```diff
@@ -36,21 +37,23 @@
 def upgrade():
     """Upgrade ill request operation logs records.
 
-    For all ON_LOAN records, we will add a new `checkout_location_pid` field
-    used to calculate fees based on checkout library.
+    For all ill request operation logs, we will add a library pid.
     """
-    query = RecordsSearch(index=LoanOperationLog.index_name)\
-        .filter('term', record__type='illr')
+    query = RecordsSearch(index=LoanOperationLog.index_name) \
+        .filter('term', record__type='illr') \
+        .filter('bool', must_not=[
```

Comment:
use `exclude()` seems more compact and more readable
```python
query =  RecordsSearch(index=LoanOperationLog.index_name) \
    .filter('term', record__type='illr') \
    .exclude('exists', field='ill_request.library_pid') \
    .source(['pid'])
```

Reply:
Thanks, changed.